charts/dvpe-deployment-gloo/values.yaml

```diff
@@ -194,38 +194,68 @@ gloo:
           scopes:
           # gloo.authConfig.spec.configs.oauth.cookie_domain -- The domain to be used for `id_token` and `access_token` cookies set after successful authentication. This has to be some kind of wildcard to support cross origin requests. If unset, the cookies get no domain set.
           cookie_domain:
```

Comment:
Can you change all variables to camleCase? We should use this breaking change to fix it

Reply:
done